sonaranalyzer-dotnet/its/regression-test.ps1

```diff
@@ -96,4 +196,12 @@ finally {
     Write-Host "Removing the import before target file from '${msBuildImportBefore}'"
     Remove-Item -Force (Join-Path $msBuildImportBefore "\SonarAnalyzer.Testing.ImportBefore.targets") `
         -ErrorAction Ignore
+
+    $sw.Stop()
```

Comment:
Is it meant to stay there or was it only for test purposes?

Reply:
I prefer to keep all of them, so if we have any slowness in the future, the perf data is already there.

Reply:
Ok to keep them all but then start using a proper variable name. `sw`, `timer1` and so on don't help understanding what they measure.